packages/shared/src/lib/core/nfts/constants/default-nft-filter.constant.ts

```diff
@@ -0,0 +1,10 @@
+import { INftFilter } from '../interfaces/nft-filter.interface'
```

Comment:
Add nft-filter to index file of the interfaces folder and adjust code:

```suggestion
import { INftFilter } from '../interfaces'
```